src/signal.c

```diff
@@ -120,7 +120,7 @@ gnss_signal_t sid_from_constellation_index(enum constellation constellation,
  */
 u16 sid_to_global_index(gnss_signal_t sid)
 {
-  assert(code_supported(sid.code));
```

Comment:
I think this change is incorrect. `sid_to_code_index()` asserts that the SID is valid, and the global index is only defined for supported codes.